src/event-topics/request/v4.request-events-ea.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Concur Request Event
+layout: reference
+---
+
+# Concur Request Event
+
+## <a name="overview"></a>Overview
+
+This page will host the documentation for all the Concur Request related events that are publicly available on the `public.concur.request` topic in the Event Subscription Service (ESS). Subscribers will be able to subscribe to these events by following the steps described on the [Event Subscription Service v4](/api-reference/ess/v4.event-subscription.html) page.
+
+Concur Request related events are relevant for applications that are interacting with travelers before they book their trip, such as applications that context to the traveler regarding company policy, preferences, or general compliance requirements for booking travel.
+
+## <a name="limitations"></a>Limitations
+
+This event subscription is only available to partners who have been granted access by SAP Concur. Access to this documentation does not provide access to the subscription.
+
+## <a name="scope-usage"></a>Scope Usage
+
+|Name|Description|
+|---|---|
+|`travelrequest.write`|Read and write Requests.|
+
+## <a name="events"></a>Events
+
+This section lists all the publicly available events.
+
+### <a name="status-changed-event"></a>Request Status Changed Event
+
+The `statusChanged` event is intended to publish a message whenever the status of a Request changes as it traverses the approval workflow during its life cycle. Subscribers of the event are able to leverage the data in the message to perform synchronous or asynchronous actions depending on their business processes. For example, this could trigger an application to make API calls to Concur Request or start of a process in a third party application.
+
+The `statusChanged` event will always be published for the following status changes without any additional configuration:
+
+* Whenever a Request is submitted.
+* Whenever a Request is sent back to the employee by an approver/processor.
+* Whenever a Request is recalled by the employee.
+* Whenever a Request is closed.
+* Whenever a Request is reopened.
+* Whenever a Request is cancelled.
+* Whenever a Request is approved.
+* Whenever a Request stops in the legacy external validation step that is. configured using the application connector.
+
+In addition to the above, any workflow step configured for notifications by the administrator will also be published. Please refer to the **Procedure: Creating External Validations and Workflow Event Notifications** section of the [Workflow General - Setup Guide](https://www.concurtraining.com/customers/tech_pubs/Docs/_Current/SG_Req/Req_SG_Workflows.pdf) for more information on how the events can be configured and optimized.
+
+## <a name="schema"></a>Schema
+
+### <a name="status-changed-event-payload-schema"></a>statusChangd Event Payload Schema
+
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`id`|`string`|`UUID`|Randomly generated unique identifier of this the event.|
```

Comment:
not sure if it's a good phrase ("this the"). "Randomly generated unique identifier of this the event."

Reply:
fixed